Cargo.lock

```diff
@@ -3201,6 +3153,15 @@ version = "2.3.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "47be2f14c678be2fdcab04ab1171db51b2762ce6f0a8ee87c8dd4a04ed216135"
 
+[[package]]
+name = "itertools"
+version = "0.8.2"
```

Comment:
It is possible to remove this?

Reply:
I am not sure but I guess your purpose is to not have duplicated packages in lock file. However, I don't think manipulating the lock file by hand is a good idea. We were using 3 different versions of itertools, I updated them all to version `0.10`. Now we will only have one itertools in the lockfile.

Here is the diff of lockfile after the version update.
```diff
- [[package]]
- name = "itertools"
- version = "0.8.2"
- source = "registry+https://github.com/rust-lang/crates.io-index"
- checksum = "f56a2d0bc861f9165be4eb3442afd3c236d8a98afd426f65d92324ae1091a484"
- dependencies = [
- "either",
- ]

- [[package]]
- name = "itertools"
- version = "0.9.0"
- source = "registry+https://github.com/rust-lang/crates.io-index"
- checksum = "284f18f85651fe11e8a991b2adb42cb078325c996ed026d994719efcfca1d54b"
- dependencies = [
- "either",
- ]

- [[package]]
- name = "itertools"
- version = "0.10.3"
- source = "registry+https://github.com/rust-lang/crates.io-index"
- checksum = "a9a9d19fa1e79b6215ff29b9d6880b706147f16e9b1dbb1e4e5947b5b02bc5e3"
- dependencies = [
- "either",
- ]

+ [[package]]
+ name = "itertools"
+ version = "0.10.3"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+ checksum = "a9a9d19fa1e79b6215ff29b9d6880b706147f16e9b1dbb1e4e5947b5b02bc5e3"
+ dependencies = [
+ "either",
+ ]
```

Reply:
> However, I don't think manipulating the lock file by hand is a good idea. 

I didn't mean manipulating by hand, I expected that you would discover what crate is dependent on `itertools 0.8.2` and update it there, which you actually did :slightly_smiling_face: 

> I am not sure but I guess your purpose is to not have duplicated packages in lock file

Yes, and also the diff looked like `0.8.2` is newly added, which would require additional security review.